### PR TITLE
✨ Add recipe card overview 🧁 ✨

### PR DESCRIPTION
**Pull request by lisamittmann**
- add new class `RecipeCardDetails` to pass reduced recipe information to recipe overview
- add new endpoint and service method to retrieve recipes from db and return `RecipeCardDetails`
- add new page for recipe overview
- add new component for recipe card (incl. additional component for image with overlay) 
- import Google fonts 
- add page header component

### CODE REVIEW

File: frontend/src/components/PageHeader.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    )
+}
+
+const HeaderWrapper = styled.section`
+  background: var(--color-turquoise-green);
+  font-family: Halant;
+  font-size: 30px;
+  padding: 20px 20px 20px 45px;
+  color: var(--color-cultured)
+`
```
Comment (christophersiem) [inline frontend/src/components/PageHeader.js]: I would make the section to a header element and simply return the styled header by default
```suggestion
import styled from 'styled-components/macro'

export default styled.header`
  background: var(--color-turquoise-green);
  font-family: Halant;
  font-size: 30px;
  padding: 20px 20px 20px 45px;
  color: var(--color-cultured)
`
```

File: backend/src/test/java/de/neuefische/teddyscoronadiaries/controller/RecipeControllerTest.java [hunk 2]
```diff
@@ -26,24 +27,24 @@
...
     @Autowired
     private RecipeMongoDb recipeMongoDb;
 
     @Autowired
-    private TestRestTemplate restTemplate;
+    private TestRestTemplate testRestTemplate;
 
     @BeforeEach
-    public void setUp(){
+    public void setUp() {
```
Comment (christophersiem) [inline backend/src/test/java/de/neuefische/teddyscoronadiaries/controller/RecipeControllerTest.java]:
```suggestion
    public void setup() {
```

File: frontend/src/components/RecipeCardComponents/RecipeCard.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  margin: 2.5px;
+
+  p {
+    padding-left: 12px;
+    padding-right: 12px;
+    height: 60px;
+    font-size: 16px;
+    margin: 0px;
+    display: flex;
+    align-items: center;
```
Comment (christophersiem) [inline frontend/src/components/RecipeCardComponents/RecipeCard.js:29]: consider adding some standard sizes to your Globalstyles

**Review by christophersiem — APPROVED** [review]
Thanks for being hungry now 😋 Looks very good, only left 2-3 comments

File: frontend/src/components/PageHeader.js (hunk 1 above)
Comment (lisamittmann) [reply frontend/src/components/PageHeader.js]: but how would I pass the header text if the component is only a styled component? I intend on using it again later on with different header texts

Comment (lisamittmann) [reply frontend/src/components/PageHeader.js]: got it - no self-closing tag 🤣  that's what you get for working late

**Pull request merged**